python/cudf/cudf/core/series.py

```diff
@@ -2115,124 +2113,6 @@ def apply(self, func, convert_dtype=True, args=(), **kwargs):
         result.name = self.name
         return result
 
-    @_cudf_nvtx_annotate
-    def applymap(self, udf, out_dtype=None):
```

Comment:
@brandon-b-miller Can you compare this docstring to `apply` and check whether we need to copy any relevant limitations from `Series.applymap` to `Series.apply`? I don't know the execution well enough to say which parts are needed, but this docstring contains a lot of information and I don't want to lose the important bits.

Reply:
The note about not being able to reference globals other than scalar objects could arguably be moved. The rest of the restrictions are called out in the UDF notebook that `Series.apply` and `DataFrame.apply` link out to [here](https://github.com/rapidsai/cudf/blob/branch-22.08/python/cudf/cudf/core/series.py#L2032) and [here](https://github.com/rapidsai/cudf/blob/branch-22.08/python/cudf/cudf/core/dataframe.py#L3690). 

It occurs to me that we might be missing some operators in `apply`, let me look into that side of things before merging.

Reply:
@brandon-b-miller says we are not missing operators in `apply` and this can be resolved.